source/Mission.cpp

```diff
@@ -177,6 +179,28 @@ void Mission::Load(const DataNode &node)
 						" They are now mission-level properties:");
 			}
 		}
+		else if(child.Token(0) == "outfit" && child.Size() >= 3)
+		{
+			if(!outfitStr.empty())
+				node.PrintTrace("Warning: more than one outfit objective, only last one will be used:");
```

Comment:
Overwriting is the default expected behavior you dont need this warning.
The one below that warns of both being defined is good though but this one or the one in outfitter can be removed